lib/BackgroundJobs/InitialCrawlJob.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/*
+ * Copyright (c) 2021. The Nextcloud Recognize contributors.
+ *
+ * This file is licensed under the Affero General Public License version 3 or later. See the COPYING file.
+ */
+
+namespace OCA\Recognize\BackgroundJobs;
+
+use OCA\Recognize\Service\FileCrawlerService;
+use OCA\Recognize\Service\Logger;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\BackgroundJob\TimedJob;
+use OCP\IConfig;
+use OCP\IUser;
+use OCP\IUserManager;
+use Psr\Log\LoggerInterface;
+
+class InitialCrawlJob extends TimedJob {
+	public const INTERVAL = 30 * 60; // 30 minutes
+
+
+	private LoggerInterface $logger;
+
+	private IConfig $config;
+
+	private FileCrawlerService $fileCrawler;
+
+	private IUserManager $userManager;
+
+
```

Comment:
```suggestion

	private LoggerInterface $logger;
	private IConfig $config;
	private FileCrawlerService $fileCrawler;
	private IUserManager $userManager;

```

A bit less whitespace